e2e-tests/tfm/cypress/e2e/journeys/case/deal-AIN.spec.js

```diff
@@ -87,4 +87,21 @@ context('User can view a case deal', () => {
       cy.url().should('eq', relative(`/case/${dealId}/facility/${facilityId}`));
     });
   });
+
+  describe('eligibility criteria', () => {
+    it('should show the correct passed/failed criteria', () => {
+      const { eligibilityCriteriaTable } = pages.caseDealPage;
+
+      MOCK_DEAL_AIN.eligibility.criteria.forEach(({ id, answer }, index) => {
+        eligibilityCriteriaTable
+          .row(index + 1)
+          .heading(id)
+          .should('exist');
+        eligibilityCriteriaTable
+          .row(index + 1)
+          .answerTag()
+          .contains(answer ? 'Passed' : 'Failed');
```

Comment:
```suggestion
          .should('contain', answer ? 'Passed' : 'Failed');
```

The difference here is small and a bit subtle, but
- `should('contain', ...)` is an assertion
- `.contains(...)` is a selector (it'll still fail if the target is not contained in the current element, but the error will be less specific)